pallets/portfolio/src/lib.rs

```diff
@@ -97,6 +97,9 @@ decl_storage! {
         pub PortfolioLockedAssets get(fn locked_assets):
             double_map hasher(twox_64_concat) PortfolioId, hasher(blake2_128_concat) Ticker =>
             T::Balance;
+        /// Tracks all the portfolios in custody of a particular identity. Only used by the UIs.
```

Comment:
The value doesn't actually matter. Either the key will be present or it will be absent. We don't need to check the value.

i.e. There will never be a case when a key returns false.

Reply:
Yeah sure, you could just as well store a `usize`, but `()` would not be OK, because the default would mean that it's not there. Basically you need a type able to store at least one bit of info. Still, I think its a useful clarification of what the semantics of the map is, because `(k1, k2) -> bool` is a funny-looking encoding of basically a set. Indeed, I had to read twice to realize what the map was doing.